types/entities.ts

```diff
@@ -38,6 +38,33 @@ export interface User {
   bioHtml: string | null;
 }
 
+export interface SeriesWorkSummary
+  extends Omit<
+    WorkSummary,
+    "category" | "publishedAt" | "rating" | "tags" | "stats" | "locked"
+  > {
+  tags: Omit<WorkSummary["tags"], "warnings">;
+  stats: Omit<WorkSummary["stats"], "comments">;
+}
+
+export interface Series {
+  id: string;
+  name: string;
+  begunAt: string;
+  updatedAt: string;
+  creators: WorkSummary["authors"];
+  description: string | null;
+  notes: string | null;
+  words: number;
+  bookmarks: number;
+  complete: boolean;
+
+  workCount: number;
+  works: SeriesWorkSummary[];
```

Comment:
As in the other PR, I would avoid duplicate data. I would add "url" into `SeriesWorkSummary` and let people map to get title themselves and use `.length` for the count.

Reply:
Why add `url`? Works in a series don't have URLs like collections.

I will remove the duplicate data though.

Reply:
Might be confusing myself, but `workUrls` is: `workUrls: seriesWorks.map((work) => getWorkUrl({ workId: work.id }))`, so my suggestion was to have the `works` array contain each work's `url` as opposed to having it as a separate field.

Reply:
So don't have the works in the `works` array and get rid of the `SeriesWorkSummary` interface?

Reply:
No i mean add the URL in the SeriesWorkSummary interface. You should be able to get put it there, or am I missing something?

Reply:
I can do that. I was just confused because I misread your suggestion.